clients/destination.go

```diff
@@ -295,6 +299,10 @@ func (c *DestinationClient) Write2(ctx context.Context, tables schema.Tables, so
 		if err := saveClient.Send(&pb.Write2_Request{
 			Resource: resource,
 		}); err != nil {
+			if err == io.EOF {
```

Comment:
```suggestion
			if errors.Is(err, io.EOF) {
```
`errors.Is` is much more reliable as the errors are wrapped more and more often.

Reply:
I don't think we need to be overly defensive in this case, as checking `== io.EOF` is the officially documented way to check if the stream has ended, see https://grpc.io/docs/languages/go/basics/#client-side-streaming-rpc